app/src/main/java/com/example/assistgoandroid/Contact/deleteContactDialog.java

```diff
@@ -0,0 +1,49 @@
+package com.example.assistgoandroid.Contact;
+
+import android.app.AlertDialog;
+import android.app.Dialog;
+import android.content.Context;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import androidx.annotation.NonNull;
+import androidx.appcompat.app.AppCompatDialogFragment;
+import com.example.assistgoandroid.R;
+
+//https://www.youtube.com/watch?v=ARezg1D9Zd0&t=633s
```

Comment:
Do you need to include this link here? It's already in the readme under the resource section.